tools/clang/test/CodeGenHLSL/d3dcolortoubyte4/d3dcolortoubyte4-const-input-fxc-compat.hlsl

```diff
@@ -0,0 +1,16 @@
+// RUN: %dxc -HV 2016 -E main -T vs_6_0 %s | FileCheck %s
```

Comment:
Nothing runs this test. Can you put it in `batch/expressions/intrinsics/`? `d3dcolortoubyte4` subfolder at your taste.

Reply:
done.